source/MaterialXGenShader/Util.cpp

```diff
@@ -699,101 +699,4 @@ ValueElementPtr findNodeDefChild(const string& path, DocumentPtr doc, const stri
     return valueElement;
 }
 
-unsigned int getUIProperties(const ValueElementPtr nodeDefElement, UIProperties& uiProperties)
```

Comment:
Figured this really doesn't belong here so moved to render utils.

Reply:
Excellent